zerver/views/registration.py

```diff
@@ -119,6 +119,11 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
     name_validated = False
     full_name = None
     require_ldap_password = False
+    billing_enabled = False
+    organization_type = ""
+
+    if settings.BILLING_ENABLED:
+        billing_enabled = True
```

Comment:
We don't need to this kind of thing. Best to just use settings.BILLING_ENABLED directly.

Reply:
I had done it previously to maintain the consistency with `require_ldap_password` as it is implemented in that way. I had made the change though.

Reply:
> I had done it previously to maintain the consistency with `require_ldap_password` 

Ah, I see! The reason for require_ldap_password being that way is that the decision whether it should be True or False is much more complicated than just reading a setting :slightly_smiling_face:

Reply:
Ok, got it. Thanks